packages/react-native-codegen/android/build.gradle

```diff
@@ -42,5 +43,25 @@ task('buildCodegenCLI', type: Exec) {
     nodeModulesDir.mkdirs();
     outputs.dirs(libDir, nodeModulesDir)
 
-    commandLine("$codegenRoot/scripts/oss/build.sh")
+    if (Os.isFamily(Os.FAMILY_WINDOWS)) {
+      // Convert path to Linux format: use canonical path to strip it off relative elements in the middle of the string.
+      // Then replace baskslashes with slashes, remove leading colon, add leading slash.
+      // Eg. D:\path1\sub2/.. -> /D/path1/path2
+      String canonicalPath = new File(codegenRoot).getCanonicalPath()
+      String linuxPath = canonicalPath.replace('\\', '/');
+      linuxPath = linuxPath.replace(':', '')
+      linuxPath = '/' + linuxPath
+      codegenRoot = linuxPath
```

Comment:
Why update `codegenRoot` if it looks like we're not using it?

Reply:
Oh right, I'm using another variable there, will remove